### ISSUE TITLE
Issue reading SVHN dataset on Windows

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#6017]
I've run run the YOLO house number detection example recently on Linux (with snapshots), and it ran fine.

Now, when I try to run it on Windows, I get the following on both the train and the test sets:
```
Exception in thread "main" java.lang.RuntimeException: H5Dread failed
	at org.bytedeco.javacpp.hdf5$DataSet.read(Native Method)
	at org.datavec.image.recordreader.objdetect.impl.SvhnLabelProvider.<init>(SvhnLabelProvider.java:76)
	at org.deeplearning4j.examples.convolution.objectdetection.HouseNumberDetection.main(HouseNumberDetection.java:96)
HDF5-DIAG: Error detected in HDF5 (1.10.2) thread 0:
  #000: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5Dio.c line 216 in H5Dread(): can't read data
    major: Dataset
    minor: Read failed
  #001: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5Dio.c line 587 in H5D__read(): can't read data
    major: Dataset
    minor: Read failed
  #002: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5Dchunk.c line 2304 in H5D__chunk_read(): unable to read raw data chunk
    major: Low-level I/O
    minor: Read failed
  #003: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5Dchunk.c line 3659 in H5D__chunk_lock(): data pipeline read failed
    major: Data filters
    minor: Filter operation failed
  #004: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5Z.c line 1254 in H5Z_pipeline(): required filter 'deflate' is not registered
    major: Data filters
    minor: Read failed
  #005: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5PLint.c line 270 in H5PL_load(): search in path table failed
    major: Plugin for dynamically loaded library
    minor: Can't get value
  #006: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5PLpath.c line 604 in H5PL__find_plugin_in_path_table(): search in path C:\ProgramData\hdf5\lib\plugin encountered an error
    major: Plugin for dynamically loaded library
    minor: Can't get value
  #007: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5PLpath.c line 734 in H5PL__find_plugin_in_path(): can't open directory
    major: Plugin for dynamically loaded library
    minor: Can't open directory or file
```

The path looks correct, and changing the separators to all ```/``` or ```\``` doesn't make a difference...

Edit: seems to me that the problem isn't the actual dataset at all - rather, it's an issue with the HDF5 library not being able to scan for plugins??
```H5PL__find_plugin_in_path(): can't open directory```

https://support.hdfgroup.org/ftp///HDF5/prev-releases/hdf5-1.10/hdf5-1.10.2/src/unpacked/src/H5PLpath.c
```
/*-------------------------------------------------------------------------
 * Function:    H5PL__find_plugin_in_path
 *
 * Purpose:     Given a path, this function opens the directory and envokes
 *              another function to go through all files to find the right
 *              plugin library. Two function definitions are for Unix and
 *              Windows.
```

**Comment by saudet** [comment]
Hum, looks like the compression algorithm isn't getting built on Windows...

**Comment by AlexDBlack** [comment]
So, yes it's plugin related...
```
search in path C:\ProgramData\hdf5\lib\plugin encountered an error
```

Creating this directory doesn't solve the problem.
Nor does overriding the ```HDF5_PLUGIN_PATH``` environment variable :/

```
  #006: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5PLpath.c line 604 in H5PL__find_plugin_in_path_table(): search in path C:\Temp\HDF5 encountered an error
    major: Plugin for dynamically loaded library
    minor: Can't get value
  #007: C:\projects\javacpp-presets\hdf5\cppbuild\windows-x86_64\hdf5-1.10.2\src\H5PLpath.c line 734 in H5PL__find_plugin_in_path(): can't open directory
    major: Plugin for dynamically loaded library
    minor: Can't open directory or file
```

**Comment by saudet** [comment]
That's now fixed with the JavaCPP Presets for HDF5 1.10.3-1.4.3.
